bin/ember-template-lint.js

```diff
@@ -265,7 +260,7 @@ function printPending(results, options) {
   }
   let pendingListString = JSON.stringify(pendingList, null, 2);
 
-  if (options.json) {
+  if (options.format === 'json') {
```

Comment:
This is a bug fix. Previously, `--print-pending` only worked with `--json` but not `--format=json`.

Reply:
CC: @scalvert

Reply:
Thank you.

Reply:
Extracting the fix to master: https://github.com/ember-template-lint/ember-template-lint/pull/2210

Although it turns out both `--json` and `--print-pending` will be removed in v4.